containersQa/testlib.bash

```diff
@@ -924,4 +928,37 @@ function newUserCheck() {
 EOF
     $PD_PROVIDER build -f $podmanfile .
 
-}
\ No newline at end of file
+}
+
+function setAlgorithmTestsUrlVars {
+  checkAlgorithmsUrl="https://raw.githubusercontent.com/rh-openjdk/jtreg-buffer/ea3c06d3d67cb05/test/reproducers/checkAlgorithms/CheckAlgorithms.java"
```

Comment:
Why to wget excat revision? I think the current tip of the files would be ideal this time, right?

Reply:
I used this revision (which is now the current head), so if there are some breaking changes to the files, the tests here still pass. I can change it to main if you want.

Reply:
hmm. Thats a good reasoning, but we know many other suites are using those files, right? So we can not do breaking changes. If breaking changes will be planed, this can be always frozen. For now, I would pick up floating tip.

Reply:
Ok, I will change it.